### PR TITLE
Housing subcategory

### PR DESCRIPTION
**Pull request by danni**

### CODE REVIEW

File: src/pages/personalisation/BaseMultiQuestion.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+                            <mui.Checkbox
+                                checked={selected.has(answer)}
+                                checkedIcon={<icons.CheckboxSelected />}
+                                unCheckedIcon={<icons.CheckboxUnselected />}
+                                onCheck={this.onAnswerTouchTap.bind(
+                                    this, answer, !selected.has(answer)
+                                )}
+                            />
+                        }
+                        disableFocusRipple={true}
```
Comment (DanielHeath) [inline src/pages/personalisation/BaseMultiQuestion.js:96]: can we extract that into a wrapper for `mui.<thing>`

File: src/pages/personalisation/BaseMultiQuestion.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+    render(): React.Element {
+        var selected = this.state.answers;
+
+        return (
+            <div>
+                <components.HeaderBar
+                    primaryText={
+                        <div>
+                            <icons.LogoLight className="Logo" />
```
Comment (DanielHeath) [inline src/pages/personalisation/BaseMultiQuestion.js]: Shouldn't need the className anymore

File: test/features/10-functional/20-personalisation/80-change-what-you-need.feature [hunk 3]
```diff
@@ -8,6 +8,7 @@ Feature: Change your personalisation settings
     Background:
         Given my location is "Melbourne VIC"
         And I have somewhere to sleep tonight
+        And I need nothing for housing
```
Comment (DanielHeath) [inline test/features/10-functional/20-personalisation/80-change-what-you-need.feature:11]: The intent behind that isn't the clearest to me

Comment (DanielHeath) [inline test/features/10-functional/20-personalisation/80-change-what-you-need.feature:11]: maybe "I don't select any specific needs for housing"

**Comment by DanielHeath** [comment]
Overall looks good, ++ with those comments

File: src/pages/personalisation/BaseMultiQuestion.js (hunk 1 above)
Comment (danni) [inline src/pages/personalisation/BaseMultiQuestion.js:96]: The focus ripples?

Comment (DanielHeath) [inline src/pages/personalisation/BaseMultiQuestion.js:96]: Yeah. No biggie if we don't though.

File: test/features/10-functional/20-personalisation/80-change-what-you-need.feature (hunk 3 above)
Comment (danni) [inline test/features/10-functional/20-personalisation/80-change-what-you-need.feature:11]: "I don't have any specific housing needs" ?

Comment (danni) [inline test/features/10-functional/20-personalisation/80-change-what-you-need.feature:11]: I'm struggling to think of something I like.

File: src/pages/personalisation/BaseMultiQuestion.js (hunk 1 above)
Comment (danni) [inline src/pages/personalisation/BaseMultiQuestion.js:96]: Yeah, I want to, every single List and ListItem have the same set of overrides. I'll abstract it in a branch tomorrow.

**Comment by danni** [comment]
Merging this for demo with 2 items on notice.

**Pull request merged**